shells/firefox/panel/run.js

```diff
@@ -21,10 +23,21 @@ var node = document.getElementById('container');
 ReactDOM.render(<h2 id="message">Looking for React...</h2>, node);
 var port = {};
 
-// TODO (bvaughn) Read default :themeName and switch between 'FirefoxLight' and 'FirefoxDark'
-// Refer to 'shells/chrome/src/panel.js' for an example
 const browserName = 'Firefox';
-const themeName = 'FirefoxLight';
+let themeName = 'FirefoxLight';
+
+// If available, mirror the current Firefox devtools theme:
+// https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/devtools.panels/themeName
+if (typeof devtools !== 'undefined' && devtools.panels) {
+  switch (devtools.panels.themeName) {
```

Comment:
I'm now wondering if this is because this code is an add-on sdk environment and that API is part of the WebExtensions.  `browser.devtools.panels.themeName` should work if you load the Chrome WebExtension into Firefox.